firefox-ios/firefox-ios-tests/Tests/XCUITests/SearchTest.swift

```diff
@@ -276,6 +277,7 @@ class SearchTests: BaseTestCase {
         validateSearchSuggestionText(typeText: "localhost")
         restartInBackground()
         // Open new tab
+        mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Browser.UrlBar.cancelButton])
```

Comment:
We may need to add a timeout here to make this even more stable

Reply:
updated